src/frontend/src/tables/settings/UserTable.tsx

```diff
@@ -298,7 +306,43 @@ export function UserTable({
             setSelectedUser(record.pk);
             deleteUser.open();
           }
-        })
+        }),
+        {
+          icon: <IconUserCircle />,
+          title: t`Open Profile`,
+          onClick: () => {
+            navigate(getDetailUrl(ModelType.user, record.pk));
+          }
+        },
+        {
+          icon: <IconKey />,
+          title: t`Change Password`,
+          color: 'blue',
+          onClick: () => {
+            setSelectedUser(record.pk);
+            setPassowrd.open();
+          }
```

Comment:
This action should only be available if the user is a superuser, right?

Reply:
I have added a check to hide it; backend requires superuser